README.md

```diff
@@ -1,10 +1,139 @@
 # GenZ
-## Requirements
 
-- [Go >1.20](https://go.dev/doc/install)
-## Try it out
-Explore built-in `examples`, clone repo, and run
+[![Go Reference](https://pkg.go.dev/badge/github.com/Joffref/genz.svg)](https://pkg.go.dev/github.com/Joffref/genz)
+[![Go Report Card](https://goreportcard.com/badge/github.com/Joffref/genz)](https://goreportcard.com/report/github.com/Joffref/genz)
+[![License](https://img.shields.io/github/license/Joffref/genz)](LICENSE)
+![GitHub stars](https://img.shields.io/github/stars/Joffref/genz)
+> At this moment, this project is in development. It is not ready for production use.
+> Refers to the [Roadmap](#roadmap) section for more information on what has been done.
+>
+> See the [Contributing](#contributing) section if you would like to help make it better.
+
+
+<p align="center" width="100%">
+    <img width="50%" src="./docs/assets/logo_500_500.png"> 
+</p>
+
+GenZ is your all-in-one opinionated code generator for Go. It leverages the power of Go templates to generate code from your own templates.
```

Comment:
```suggestion
:sparkles: Never write toil/boilerplate code again! :sparkles: 
GenZ is a template-based generator for Go. A single binary that can be called with the native `go generate` to automate generation of Go tested code based on your templates.
```

(I would have put the Getting Started just after this, and installation at the end)